Sources/FluentKit/Properties/Timestamp.swift

```diff
@@ -1,5 +1,16 @@
+import class Foundation.ISO8601DateFormatter
+import class Foundation.DateFormatter
+import class NIO.ThreadSpecificVariable
+
 extension Fields {
-    public typealias Timestamp = TimestampProperty<Self>
+    public typealias Timestamp<Formatter> = TimestampProperty<Self, Formatter>
```

Comment:
Adding a new generic parameter to a public type is a breaking change. For example:

```swift
import FluentKit

func foo(_ timestamp: User.Timestamp) {
    print(timestamp.key)
}
```

This would break and `foo` would need to become generic. 

```swift
func foo<Formatter>(_ timestamp: User.Timestamp<Formatter>) { ... }
```

Reply:
We haven't officially released FluentKit, so that's technically not an issue. 😁At the same time, I agree, we should avoid this. I'll see if I can get around that.